simple_switch_grpc needs to link against libpip4info

When I follow the instructions in simple_switch_grpc's README, I get the following build failure:

```
/bin/sh ./libtool  --tag=CXX   --mode=link g++ -Wall -Wextra -pthread -g -O2 -Wl,--no-as-needed,-lgrpc++_reflection,--as-needed -rdynamic  -o simple_switch_grpc main.o libsimple_switch_grpc.la 
libtool: link: g++ -Wall -Wextra -pthread -g -O2 -Wl,--no-as-needed -Wl,-lgrpc++_reflection -Wl,--as-needed -rdynamic -o .libs/simple_switch_grpc main.o  ./.libs/libsimple_switch_grpc.a -lboost_system -lthrift -lboost_program_options -lboost_filesystem /home/bpfaff/dddp/behavioral-model/PI/.libs/libbmpi.so -lboost_thread -ldl -lpcap -lgmp -lJudy -lnanomsg /home/bpfaff/dddp/behavioral-model/targets/simple_switch_grpc/.libs/libbm_grpc_dataplane.so -lpifeproto -lpigrpcserver -lpi -L/usr/lib -lgrpc++ -L/usr/usr/lib64 -lgrpc -lgpr -lz -lcares -lssl -lcrypto -lprotobuf -lpthread -pthread -Wl,-rpath -Wl,/usr/local/lib
/usr/bin/ld: skipping incompatible /usr/lib/libdl.so when searching for -ldl
/usr/bin/ld: skipping incompatible /usr/lib/libpthread.so when searching for -lpthread
/usr/bin/ld: skipping incompatible /usr/lib/libm.so when searching for -lm
/usr/bin/ld: skipping incompatible /usr/lib/libpthread.so when searching for -lpthread
/usr/bin/ld: skipping incompatible /usr/lib/libc.so when searching for -lc
/usr/bin/ld: ./.libs/libsimple_switch_grpc.a(switch_runner.o): undefined reference to symbol 'pi_p4info_table_id_from_name'
/usr/bin/ld: /usr/local/lib/libpip4info.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

I added ``-lpip4info`` to ``simple_switch_grpc_LDADD`` in ``targets/simple_switch_grpc/Makefile.am`` and that resolved the issue. (I doubt this is the preferable resolution, probably ``configure`` should arrange for this.)

--- a/targets/simple_switch_grpc/Makefile.am
+++ b/targets/simple_switch_grpc/Makefile.am
@@ -53,7 +53,7 @@ $(builddir)/../simple_switch/libsimpleswitch_thrift.la
 endif
 
 libsimple_switch_grpc_la_LIBADD += \
--lpifeproto -lpigrpcserver -lpi \
+-lpifeproto -lpigrpcserver -lpi -lpip4info \
 $(GRPC_LIBS) $(PROTOBUF_LIBS)
 
 # dataplane_interface.proto

